src/components/navbar/index.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
```

Comment:
I am on the fence over this one. Why are we importing everything? shouldn't you import only the bits you need?

Reply:
![image](https://user-images.githubusercontent.com/83610935/204027912-b0a3cb03-a6e2-405c-bd6c-7a707ed16cf2.png)
It gives me this error when i fail to import. And still I'm going to use the import once i get the design for the mobile version of the navbar